six/modules/c++/six.sicd/include/six/sicd/GeoData.h

```diff
@@ -98,6 +99,8 @@ class GeoData
     GeoData() :
         earthModel(EarthModelType::WGS84)
     {
+        scp.ecf = Init::undefined<Vector3>();
```

Comment:
I’m having dependency issues when trying to include Init.h in Types.h, since the former relies on a lot of things the latter defines. Should I just set it manually to the value? Or I could start a Types.cpp.

There were similar issues with including struct header files in each other, vs. forward declaring, though I don’t recall the specifics. I can at least try to include what I can there.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Sunday, August 21, 2016 5:26 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [ngageoint/six-library] Sicd validation (#91)


In six/modules/c++/six.sicd/include/six/sicd/GeoData.h<https://github.com/ngageoint/six-library/pull/91#discussion_r75603695>:

> @@ -98,6 +99,8 @@ class GeoData

>      GeoData() :

>          earthModel(EarthModelType::WGS84)

>      {

> +        scp.ecf = Init::undefined<Vector3>();

Update SCP's constructor rather than doing that here

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ngageoint/six-library/pull/91/files/550f20e29ec33ccccbba2937926fc0491d89b757#r75603695>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFRAYNktaP2bbq8xEZcsYyRLq6fEmHxHks5qiMJggaJpZM4JO-9v>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.